ingress/README.md

```diff
@@ -75,15 +75,17 @@ resources:
 
 ```bash
 VOXEL51_API_TOKEN=/path/to/your/api-token.json
-ANALYTIC_NAMES='voxel51/person-sense,voxel51/road-sense,voxel51/vehicle-sense' # optional
+ANALYTIC_NAMES='voxel51/person-sense,voxel51/road-sense,voxel51/vehicle-sense' # Optional
+EXPIRATION_SECONDS=259200 # Optional (sets ttl of data and job)
```

Comment:
support individual configuration of `DATA_EXPIRATION_SECONDS` and `JOB_EXPIRATION_SECONDS`?

Reply:
Delta between these would not allow them to preview the output after the data expires.